index.bs

```diff
@@ -613,7 +613,7 @@ Unlike WebGPU, this API does not intrinsically support custom shader authoring;
 
 The WebGPU API identifies <a href="https://gpuweb.github.io/gpuweb/#privacy-machine-artifacts">machine-specific artifacts</a> as a privacy consideration. Similarly, the WebNN API's compute unit scheduling may under certain circumstances introduce a fingerprint. However, similarly to WebGPU, such fingerprints are identical across most or all of the devices of each vendor, mitigating the concern. Furthermore, software implementations can be used to further eliminate such artifacts.
 
-The WebNN API defines two developer-settable preferences to help inform [[#programming-model-device-selection]] and allow the implementation to better select the most appropriate underlying execution device for the workload. An {{MLDeviceType}} normatively indicates the kind of device and is either {{MLDeviceType/"cpu"}} or {{MLDeviceType/"gpu"}}. If this type cannot be satisfied, an "{{OperationError}}" {{DOMException}} is thrown, thus this type can in some cases add two bits of entropy to the fingerprint. An {{MLPowerPreference}} indicates preference as related to the power consumption and is considered a hint only and as such does not increase entropy of the fingerprint.
+The WebNN API defines two developer-settable preferences to help inform [[#programming-model-device-selection]] and allow the implementation to better select the most appropriate underlying execution device for the workload. An {{MLDeviceType}} normatively indicates the kind of device and is one of: {{MLDeviceType/"cpu"}}, {{MLDeviceType/"gpu"}}, {{MLDeviceType/"npu"}}. If this type cannot be satisfied, an "{{OperationError}}" {{DOMException}} is thrown, thus this type can in some cases add two bits of entropy to the fingerprint. An {{MLPowerPreference}} indicates preference as related to the power consumption and is considered a hint only and as such does not increase entropy of the fingerprint.
```

Comment:
Similar to MLContextOptions being under active development as called out on line 729 below, I think we should refer to https://github.com/webmachinelearning/webnn/issues/623 here as well and state that MLDeviceType is subject to change.

Reply:
Are you wanting the whole block, or a shorter sentence here? Given this is the privacy section, I don't think it needs *quite* as much verbiage as that block in the relevant `MLContextOptions` section. Maybe just `Issue(623): MLContextOptions is under active development, and the design is expected to change, informed by further implementation experience and new use cases from the wider web community.`?

Reply:
A shorter sentence is great

Reply:
> It is outside the scope of this change, but it would be preferable to remove the If this type cannot be satisfied, an "{{OperationError}}" {{DOMException}} is thrown.

Yeah, the exact error policy is certainly a question we should shore up. I'm deferring that as part of 623's "Other considerations under discussion include error handling, ultimate fallback and quantized operators".

Reply:
Thanks Mike. I'll go ahead and merge this, as I believe all feedback has been addressed now.